apps/patient/src/components/BrandedPharmacyCard.tsx

```diff
@@ -0,0 +1,67 @@
+import {
+  Card,
+  CardBody,
+  Image,
+  SlideFade,
+  Text,
+  VStack,
+  useBreakpointValue
+} from '@chakra-ui/react'
+
+// @ts-ignore
```

Comment:
I think we can get rid of these ts-ignore on images if we place a `assets.d.ts` file at the root of patient `/src`:

``` ts
// asset.d.ts
declare module '*.png' {
  const content: string
  export default content
}

declare module '*.svg' {
  const content: string
  export default content
}
```

https://graffino.com/til/hEvDjQa4au-how-to-import-images-in-typescript

Reply:
nice! 🙏

Reply:
rad site as well